frontend/src/helpers/ApiPaths.tsx

```diff
@@ -1,3 +1,4 @@
 export class ApiPaths{
     public static USER_API = '/api/users'
+    public static HOBBY_COLLECTOR_API = "/api/hobby-collectors"
 }
```

Comment:
Diese Klasse ist ganz gut zum Organisieren. :+1::raised_hands:

Reply:
Mit static in der Klasse kann man Konstanten festlegen. Über ApiPath. <- achte auf den Punkt. Und Variblennamen groß schreiben.